packages/devel/mold/package.mk

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2022 Team LibreELEC (https://libreelec.tv)
```

Comment:
```suggestion
# Copyright (C) 2022-present Team LibreELEC (https://libreelec.tv)
```
following distribution standards

Reply:
I'm not going to add that. Sorry, I've gone over this before.

Please read: https://liferay.dev/blogs/-/blogs/how-and-why-to-properly-write-copyright-statements-in-your-code

specifically:

>A special case is when people use a range like {$year}-present. This has almost all of the above-mentioned issues, plus it adds another dimension of confusion, because what constitutes the “present” is an open – and potentially philosophical – question. Does it mean:
>
> the time when the file was last modified?
> the time it was released as a package?
> the time you downloaded it (maybe for the first time)?
> the time you ran it the last time?
> or perhaps even the ever eluding “right now”?

Reply:
This is not an issue tracker, pls stop spamming your random ideas/opinions here. 

We have distribution standards, so if something gets merged it should follow the standards in beforehand to avoid some mess. This is the common practice at every bigger repository.

If you think we need to change something at these standards pls open an issue at our [issue tracker](https://github.com/LibreELEC/LibreELEC.tv/issues) and do not hijack random PRs for that.

Reply:
So you are going to block this because you think the copyright is wrong?

Reply:
You are right, switched it to WIP as it does not align with the current distribution standards and needs additional changes.

If **YOU** think there are reasons for changing our current "guidelines" then see open an [issue](https://github.com/LibreELEC/LibreELEC.tv/issues). 
Trying to change guidelines buried at some random PR is for obvious reasons the wrong approach.